### DISCUSSION TITLE
Is there anything special about the dictionary containing different types of texts? How much loss of accuracy will the diversity cause?

### DISCUSSION THREAD

**Discussion by xiaoyangyang2** [PaddlePaddle/PaddleOCR#6020] [Q&A] [answered]
Is there anything special about the dictionary containing different types of texts? How much loss of accuracy will the diversity cause?

**Comment by Evezerest** [comment, marked as answer]
It may lead to the oversized FC in the last layer and the model size increase. If you have any special requirements, you can merge dictionaries of different text types you need into one dictionary and use it to train the model. But if you introduce too many close words, there may be a loss of accuracy. And the issue of character balance also needs to be taken into account. You can separate the dicitionaries in PaddleOCR for the time being.
